measure.rb

```diff
@@ -2835,25 +2829,6 @@ def self.add_ceiling_fans(runner, model, building, weather)
                             @cfa, @living_space)
   end
 
-  def self.check_distribution_system(building, system_values)
```

Comment:
New EPvalidator will check the id reference as well as distribution system type, this test can be removed, right?

Reply:
Seems like it. 🚀